test/README.md

```diff
@@ -0,0 +1,65 @@
+# Performance Tests
+
+# Rendering Tests
+
+Tests for rendering components, which compares image result and stored reference image.
```

Comment:
Reword.
Tests for components *that* render images. They render images to canvas and compare the result image with approved reference images.